src/server/pfs/server/api_server.go

```diff
@@ -71,23 +72,23 @@ func (a *apiServer) ActivateAuth(ctx context.Context, request *pfs.ActivateAuthR
 
 func (a *apiServer) ActivateAuthInTransaction(txnCtx *txncontext.TransactionContext, request *pfs.ActivateAuthRequest) (response *pfs.ActivateAuthResponse, retErr error) {
 	// Create role bindings for projects created before auth activation
-	var projectInfo pfs.ProjectInfo
-	if err := a.driver.projects.ReadWrite(txnCtx.SqlTx).List(&projectInfo, col.DefaultOptions(), func(string) error {
+	projIter, err := coredb.ListProject(txnCtx.Context(), a.env.DB)
```

Comment:
`Context` is a method that was mistakenly added to txnCtx, and appears to have no uses in the code right now.  We should get rid of that ASAP and use a better transaction object; it should not expose the context, but should implement Select/Exec/Query/etc. and automatically pass the context in.

Reply:
Also, I don't see a ton of value in using an iterator other than sql.Rows here; there is no need to do this a page at a time as we have to read every project here.  Might as well not make multiple round-trips to the database.

Reply:
I agree with getting rid of the context in txnCtx, although I'm not sure how I feel about doing queries outside of the transaction while using a transaction object. It seems like the usage should either be clearly outside of a transaction with a context, or clearly inside a transaction with only a `sql.Tx`.